cmd/containerd-shim-runhcs-v1/pod.go

```diff
@@ -84,6 +85,16 @@ func createPod(ctx context.Context, events publisher, req *task.CreateTaskReques
 	owner := filepath.Base(os.Args[0])
 	isWCOW := oci.IsWCOW(s)
 
+	templateID := oci.ParseAnnotationsTemplateID(ctx, s)
+	isTemplateCreation := oci.ParseAnnotationsSaveAsTemplate(ctx, s)
```

Comment:
I feel like it would be easier to read if this was called something like `createTemplate`. Thoughts?